docs/design/de-skills-design-summary.md

```
@@ -0,0 +1,305 @@
+| `test_reconciliation.py` | 对数校验 SQL 生成 | 13 |
+| `test_database.py` | DBFuncTool 新方法 | ~20 |
+| `test_gen_job_agentic_node.py` | gen_job subagent | 17 |
+| `test_migration_agentic_node.py` | migration subagent | 17 |
+| `test_migration_integration.py` | 集成测试（GP+SR） | 8 |
+| `test_migration_subagent.py` | 子代理集成测试 | 6 |
+
+---
+
+## 7. 架构图
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)